pkg/apis/system/v1beta1/types.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1beta1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+const (
+	BundleNone       = "none"
+	BundleCollecting = "collecting"
+	BundleCollected  = "collected"
+)
+
+// +genclient
+// +genclient:nonNamespaced
+// +genclient:onlyVerbs=get,create,delete
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type Bundle struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Status   string `json:"status,omitempty"`
+	Sum      string `json:"sum,omitempty"`
+	FilePath string `json:"filePath,omitempty"`
```

Comment:
As we discussed in the review meeting, we should expose the download URL here. I personally feel no need to expose the file path, as it is internal implementation (we can document the bundle path/dir if people want to check the file in FS).

Reply:
The `FilePath` is needed for in pod usage, it tells users where to find the bundle.
The `Downloadlink` looks not that important IMO. If users can retrieve the resource, then they can access `/download` sub-resource easily. Meanwhile, since the download link is pointing to a sub-resource, the real link depends on the palace the REST be installed.

Reply:
For users log into the Pod, could we assume they know about the log location (as we can document it). I just feel not good to expose internal logic (in this case internal FS path) via API.

For download URL, I feel it might be more convenient for some users. But I am fine not to add it.

Reply:
Got it, I would remove the `FilePath` field.